### DISCUSSION TITLE
Better Card Organization

### DISCUSSION THREAD

**Discussion by lungdoctortn** [codersaiful/ultraaddons-elementor-lite#176] [Feature Request]
One of the most difficult aspects of website production is the card.  You have a card feature, but the feature most needed is the following:  When multiple cards are lined in a row, each with an image the same size, a title, prose, then a button.  The problem lacking in all cards is that there is no way to have automatic production of cards with the same height, buttons at the bottom all lined up horizontally across the bottom, the prose of different line lengths necessitating different sized space between the prose and the button, and finally titles with varying length maybe one or two lines which line up.

**Comment by fazlebarisn** [comment]
Can you please give an example? Screenshot or a link. I am not clear enough.
